dora/core/client/fs/src/main/java/alluxio/client/file/dora/ConsistentHashProvider.java

```diff
@@ -66,26 +68,26 @@ public class ConsistentHashProvider {
    * Used to compare with incoming worker list to skip the heavy build process if the worker
    * list has not changed.
    */
-  private final AtomicReference<List<BlockWorkerInfo>> mLastWorkerInfos =
-          new AtomicReference<>(ImmutableList.of());
+  private final AtomicReference<List<WorkerIdentity>> mLastWorkers =
+      new AtomicReference<>(ImmutableList.of());
   /**
    * Requirements for interacting with this map:
    * 1. This hash ring is lazy initialized (cannot init in the constructor).
-   *    Multiple threads may try to enter the init section and it should be only initialized once.
+   * Multiple threads may try to enter the init section and it should be only initialized once.
    * 2. This hash ring is timestamped. After the TTL expires, we need to compare the worker
-   *    list with the current available workers and possibly rebuild the hash ring.
+   * list with the current available workers and possibly rebuild the hash ring.
    * 3. While the hash ring is being updated, readers should see a stale hash ring
-   *    without blocking.
-   *
+   * without blocking.
+   * <p>
    * Thread safety guarantees:
    * 1. At lazy-init time, mutual exclusion is provided by `synchronized(mInitLock)`
-   *    and double-checking. At this stage it is guarded by `mInitLock`.
+   * and double-checking. At this stage it is guarded by `mInitLock`.
    * 2. After init, updating the hash ring is guarded by an optimistic lock using CAS(timestamp).
-   *    There will be no blocking but a read may see a stale ring.
-   *    At this stage it is guarded by `mLastUpdatedTimestamp`.
+   * There will be no blocking but a read may see a stale ring.
```

Comment:
I wonder if this is beautified automatically by some scripts or IDE?
Why such change is needed?

Reply:
Yeah, IDE auto formatting. Reverted.